packages/@adobe/spectrum-css-temp/components/breadcrumb/index.css

```diff
@@ -121,43 +121,44 @@ governing permissions and limitations under the License.
   pointer-events: none;
 }
 
-.spectrum-Breadcrumbs--compact {
+.spectrum-Breadcrumbs--small {
   .spectrum-Breadcrumbs-item {
-    font-size: var(--spectrum-breadcrumb-compact-item-text-size);
+    font-size: var(--spectrum-breadcrumb-multiline-item-text-size);
   }
 }
 
-.spectrum-Breadcrumbs--multiline {
-  flex-wrap: wrap;
-
+.spectrum-Breadcrumbs--medium {
   .spectrum-Breadcrumbs-item {
-    font-size: var(--spectrum-breadcrumb-multiline-item-text-size);
-    font-weight: var(--spectrum-breadcrumb-multiline-item-text-font-weight);
-    line-height: var(--spectrum-breadcrumb-multiline-item-text-line-height);
+    font-size: var(--spectrum-breadcrumb-compact-item-text-size);
+  }
+}
 
-    &:last-of-type {
-      height: auto;
+.spectrum-Breadcrumbs-item-multiline {
+  -webkit-appearance: none;
+  height: auto;
 
-      flex: 0 1 100%;
+  flex: 0 1 100%;
 
-      margin-top: var(--spectrum-breadcrumb-multiline-title-margin-top);
+  margin-top: var(--spectrum-breadcrumb-multiline-title-margin-top);
 
-      font-size: var(--spectrum-breadcrumb-multiline-title-text-size);
-      font-weight: var(--spectrum-breadcrumb-multiline-title-text-font-weight);
-      line-height: var(--spectrum-breadcrumb-multiline-title-text-line-height);
+  font-size: var(--spectrum-breadcrumb-multiline-title-text-size);
+  font-weight: var(--spectrum-breadcrumb-multiline-title-text-font-weight);
+  line-height: var(--spectrum-breadcrumb-multiline-title-text-line-height);
 
-      .spectrum-Breadcrumbs-itemLink {
-        height: auto;
+  .spectrum-Breadcrumbs-itemLink {
+    height: auto;
 
-        text-decoration: none;
-      }
-    }
+    text-decoration: none;
+    /* override webkit and user agent h1 */
+    margin-block-start: 0;
+    margin-block-end: 0;
+    font-size: var(--spectrum-breadcrumb-multiline-title-text-size);
```

Comment:
How are we expecting the user to customize the Heading size? Is there a headingSize prop that should be passed to Breadcrumb?